tsdb/head_test.go

```diff
@@ -1368,14 +1368,16 @@ func TestMemSeriesIsolation(t *testing.T) {
 		return -1
 	}
 
-	i := 0
+	i := 1
 	for ; i <= 1000; i++ {
 		var app storage.Appender
 		// To initialize bounds.
 		if hb.MinTime() == math.MaxInt64 {
-			app = &initAppender{head: hb, appendID: uint64(i), cleanupAppendIDsBelow: 0}
+			app = &initAppender{head: hb}
 		} else {
-			app = hb.appender(uint64(i), 0)
+			a := hb.appender()
```

Comment:
Why do you need this temporary variable?

Reply:
because init appender and appender both meet the storage.Appender interface, but I need to change manually the cleanupAppendIDsBelow which is not part of the interface.